lib/school.rb

```diff
@@ -0,0 +1,29 @@
+class School
+  attr_reader :start_time,
+              :hours_in_school_day
+
+  def initialize(start_time, hours_in_school_day)
+      @start_time = start_time
+      @hours_in_school_day = hours_in_school_day
+  end
+
+  # def start_time
+  #   @start_time = '9:00'
+  # end
+  #
+  # def hours_in_school_day
+  #   @hours_in_school_day = 7
+  # end
+
+  def student_names
```

Comment:
`@student_names` needs to be an instance variable we assign to an empty array in our `#initialize` method